content/docs/news/release_announcements/keptn-073/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Keptn 0.7.3
+weight: 97
+---
+
+Keptn 0.7.3 provides a new feature that allows invalidating a quality gate evaluation result. Besides, this release addresses minor bugs as well as UI/UX improvements. 
+
+**The key announcement of Keptn 0.7.3**:
+
+:rocket: *Invalidate a quality gate evaluation*: This feature allows a user to manually invalidate an evaluation result meaning that it will be ignored for future comparison. The original evaluation result (i.e., the `sh.keptn.events.evaluation-done` event) will not be changed or deleted and can still be fetched from the Keptn API.
+
+## Keptn Specification
+
+Implemented **Keptn spec** version: [0.1.7](https://github.com/keptn/spec/tree/0.1.7)
+
+- Added new event of type: `sh.keptn.events.evaluation.invalidated` - this event marks a quality gate evaluation result as invalid [#47](https://github.com/keptn/spec/issues/47) 
+
+## New Features
+
+<details><summary>Platform Support / Installer</summary>
+<p>
+
+- Added installation type as environment variable to Bridge [#2606](https://github.com/keptn/keptn/issues/2606)
+- Update NGNIX version to 1.19.4-alpha [#2651](https://github.com/keptn/keptn/issues/2615)
```

Comment:
Same as with the release-notes in keptn/keptn, it should be 1.19.4-alpine and issue #2651

Reply:
fixed it